lib/python/rose/opt_parse.py

```diff
@@ -665,7 +665,12 @@ class RoseOptionParser(OptionParser):
             {"action": "store_true",
              "dest": "view_mode",
              "default": False,
-             "help": "View with web browser."}]}
+             "help": "View with web browser."}],
+        "no_warn": [
+            ["--no-warn"],
+            {"action": "append",
+             "dest": "no_warn",
+             "help": "Warnings to disable."}]}
```

Comment:
I think you can add `"choice": ["version"]` here to restrict the allowed keyword.

You should also an a `"meta": "WARNING-TYPE"` here to indicate that the option has arguments.